plugins/value-container-assignment/testData/diagnostics/otherOperators.kt

```diff
@@ -0,0 +1,71 @@
+annotation class ValueContainer
+
+@ValueContainer
+data class StringProperty(var v: String) {
+    fun assign(v: String) {
+        this.v = v
+    }
+    fun assign(v: StringProperty) {
+        this.v = v.get()
+    }
+    fun get(): String = v
+}
+
+data class Task(val input: StringProperty)
+
+fun `should not effect error reporting for other operators`() {
+    val task = Task(StringProperty("Fail"))
+    val nullTask: Task? = null
+
+    // a.b += c
+    task.input <!UNRESOLVED_REFERENCE_WRONG_RECEIVER!>+=<!> StringProperty("Fail")
```

Comment:
Please check all following scenarios:
```
task.input += ""
```
1. There is no `StringProperty.plus(other: String)` and `StringProperty.plusAssign(other: String)`
2. Only `plus` exists
3. Only `plusAssign` exists
4. Both exists

Reply:
Added.